chart/values.schema.yaml

```diff
@@ -251,8 +251,59 @@ properties:
   features:
     type: object
     properties:
-      # "blobstore" doesn't have an "enabled" property
-      blobstore: {}
+      # Note: "blobstore" doesn't have an "enabled" property
+      blobstore:
+        type: object
+        oneOf:
+        - properties:
+            provider: {enum: [singleton]}
```

Comment:
Being able to have `blobstore: { provider: s3 }` here, and adding a comment in `unsupported.yaml`, might be easier on the migration due to the better message? On the other hand, doing this here means we can get an earlier failure, so I can see the appeal here too…

Reply:
I thought disallowing it in the schema is enough. That should give at least a strong enough hint to read the release notes, which should explain this.